portfolio/src/main/webapp/script.js

```diff
@@ -31,8 +31,10 @@ function addRandomQuote() {
 
 function fetchComment() {
   console.log('Fetching a comment.');
-
-  fetch('/data')
+  var x = document.getElementById('quantity');
+  console.log("this is value" + x.value);
+    fetch('/data?maxComment=' + document.getElementById('quantity').value)
+  //fetch('/data?maxComment=3')
     .then(response => response.json())
     .then((quotes) => {
       document.getElementById('quote-container').innerHTML = quotes;
```

Comment:
can you check the following for me, please?

1 Add a comment like `<script>console.log("hello world!");<script)`
2 then go to the page and load that comment
3 check that there's nothing in the javascript error console (we should see the words "<script>console.log("hello[..]." on the page, instead of having it treated like a real script

if so, then this is a security vulnerability (any arbitrary person can essentially execute scripts on someone else's page load): https://owasp.org/www-community/attacks/xss/

Reply:
I just checked - it's not treated like real script! just a comment.